frontend/public/components/edit-yaml.jsx

```diff
@@ -585,21 +590,70 @@ export const EditYAML = connect(stateToProps)(
       });
     }
 
+    toggleSidebar = () => {
+      this.setState((state) => {
+        return { showSidebar: !state.showSidebar };
+      });
+      window.dispatchEvent(new Event('sidebar_toggle'));
+    };
+
     render() {
       if (!this.props.create && !this.props.obj) {
         return <Loading />;
       }
 
-      const { connectDropTarget, isOver, canDrop } = this.props;
+      const { connectDropTarget, isOver, canDrop, create, yamlSamplesList } = this.props;
       const klass = classNames('co-file-dropzone-container', {
         'co-file-dropzone--drop-over': isOver,
       });
 
-      const { error, success, stale, yaml, height } = this.state;
-      const { create, obj, download = true, header } = this.props;
+      const { error, success, stale, yaml, height, showSidebar } = this.state;
+      const { obj, download = true, header } = this.props;
       const readOnly = this.props.readOnly || this.state.notAllowed;
       const options = { readOnly, scrollBeyondLastLine: false };
       const model = this.getModel(obj);
+      const samples = getResourceSidebarSamples(model, yamlSamplesList);
+      const showSamples = !_.isEmpty(samples) && create;
+      const linkLabel = `View Schema ${showSamples ? 'and Samples' : ''}`;
+      const sidebarLink = (
+        <div className="co-p-has-sidebar__sidebar--hidden hidden-sm hidden-xs">
+          <Button type="button" variant="link" isInline onClick={this.toggleSidebar}>
+            <InfoCircleIcon className="co-icon-space-r co-p-has-sidebar__sidebar-link-icon" />
+            {linkLabel}
+          </Button>
+        </div>
+      );
+      const shortcuts = (
+        <React.Fragment>
+          <div
+            className={classNames(
+              'yaml-editor__shortcuts',
+              showSidebar && 'yaml-editor__shortcuts--sidebar',
```

Comment:
This className doesn't have a rule defined.  I'm assuming this is unnecessary.

Reply:
Yeah, it's unnecessary now. I was using it earlier and forgot to remove it.